lib/worker.py

```diff
@@ -313,7 +315,7 @@ def run_task(self, task_id):
             raise
         return short_status
 
-    def run_loop(self, task_queue, result_queue):
+    def run_loop(self, task_queue, result_queue, fragile_retries):
```

Comment:
fragile_retries is saved in self.suite.ini, may be forward all fragile stuff through arguments is not a good idea, may be just pass the is_fragile flag?

Reply:
Right, removed fragile_retries and used self.suite.fragile_retries() where was needed. Set is_fragile here.